packages/ui-kit/src/components/IconButton/IconButton.tsx

```diff
@@ -7,15 +7,28 @@ import { Button, ButtonProps, ButtonLabel } from "../Button/Button";
 export interface IconButtonProps extends Omit<ButtonProps, "variant"> {
   icon: ReactNode;
   reverse?: boolean;
+  variant?: "bare";
 }
 
 export const IconButton: VFC<IconButtonProps> = ({
   label,
   icon,
   className,
+  variant,
   reverse,
   ...rest
 }) => {
+  if (variant === "bare") {
+    return (
+      <button
+        aria-label={typeof label === "string" ? label : undefined}
+        className={clsx(styles["bare-icon-button"], className)}
+        {...rest}>
+        {icon}
+      </button>
+    );
+  }
+
   const content = (
```

Comment:
I think this component is short and clear enough for this to be in the `label` prop `{}` in the return

Reply:
Done.